### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by adrianludwin**
/assign @rjbez17 
/assign @yiqigao217

### CODE REVIEW

**Comment by adrianludwin** [comment]
Just a big ugly typo in the _very first line of the user guide_, no biggie!

File: incubator/hnc/docs/user-guide/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # HNC User Guide
-Authors: aludwin@google.com and other contributes from wg-multitenancy
+Authors: aludwin@google.com and other contributers from wg-multitenancy
```
Comment (yiqigao217) [inline incubator/hnc/docs/user-guide/README.md:2]: contributors?

Comment (adrianludwin) [reply incubator/hnc/docs/user-guide/README.md]: Ah! Fixed, thanks!

**Pull request merged**